velox/common/file/FileSystems.cpp

```diff
@@ -80,7 +82,16 @@ folly::once_flag localFSInstantiationFlag;
 class LocalFileSystem : public FileSystem {
  public:
   explicit LocalFileSystem(std::shared_ptr<const config::ConfigBase> config)
-      : FileSystem(config) {}
+      : FileSystem(config),
+        executor_(
+            config->get<bool>(core::QueryConfig::kSpillReadAheadEnabled, true)
```

Comment:
This needs to be system option instead of per query config

Reply:
Now it only appeals to function `registerLocalFileSystem`, do you mean adding a struct FileSystemOption?
```
void registerLocalFileSystem() {
  registerFileSystem(
      LocalFileSystem::schemeMatcher(), LocalFileSystem::fileSystemGenerator());
}
```